composer.json

```diff
@@ -59,11 +59,11 @@
         "phpunit/phpunit": "^4.8|^5.0|^8.0",
         "yoast/phpunit-polyfills": "^1.0",
         "squizlabs/php_codesniffer": "3.*",
-        "phpdocumentor/reflection": "^3.0||^4.0",
-        "symfony/console": "^3.0",
+        "phpdocumentor/reflection": "^3.0||^4.0||^5.x-dev",
```

Comment:
Why are we adding support for `5.x-dev`? Is this because `4.x` of `phpdocumentor/reflection` requires `psr/log:1.0`? 

This is a monorepo, and is discouraged from being installed. So we should be able to support Monolog 3.x without having to add support for a major version (and especially a major version in `-dev`). We would do this by adding support in the `Core` library, and relying on the individual package tests (see https://github.com/googleapis/google-cloud-php/pull/5325).